examples/simple/root.re

```diff
@@ -1,3 +1,8 @@
+open ReasonJs;
+
 Js.log "Do you see this message? If you do, then things should work!";
 
-ReactDOMRe.render <Page message="Hello!" /> (ReasonJs.Document.getElementById "index");
+switch (document |> Document.getElementById "index") {
+| Some root => ReactDOMRe.render <Page message="Hello!" /> root;
+| None => assert false;
```

Comment:
Can you throw `Not_found` instead? Imo `assert false` should only be used in situations where that branch can really never happen. Also `Not_found` is for this exact situation.